examples/notebooks/beam-ml/README.md

```diff
@@ -18,27 +18,27 @@
 -->
 # ML Sample Notebooks
 
-As of Beam 2.40 users now have access to a
+Starting with the Apache Beam SDK version 2.40, users have access to a
 [RunInference](https://beam.apache.org/releases/pydoc/current/apache_beam.ml.inference.base.html#apache_beam.ml.inference.base.RunInference)
 transform.
 
-This allows inferences or predictions of on data for
-popular ML frameworks like TensorFlow, PyTorch and
+This transform allows you to make inferences or predictions on data for
+popular machine learning frameworks like TensorFlow, PyTorch, and
```

Comment:
From
predictions on data for popular machine learning frameworks 
To 
predictions and inference on data with ML models. The model handler abstracts the user from the configuration needed for specific frameworks, for example Tensorflow, PyTorch and others. A full list available here...

Reply:
Fixed